protocol-dashboard/src/components/UpdateDelegationModal/UpdateDelegationModal.tsx

```diff
@@ -47,149 +69,269 @@ type UpdateDelegationModalProps = OwnProps
+                  setInputValue(
+                    isIncrease
+                      ? formatWeiNumber(maxIncrease)
+                      : formatWeiNumber(maxDecrease)
+                  )
+                }}
+                className={styles.maxButton}
+                text={messages.max.toUpperCase()}
+                type={ButtonType.PRIMARY_ALT}
               />
-            </div>
-            <div className={styles.stakingRow}>
-              <div className={styles.stakingLabel}>Change:</div>
-              <DisplayAudio
-                className={clsx(styles.stakingValue, styles.changeValue)}
-                amount={stakeChange}
-                label={TICKER}
+            </Flex>
+            <Flex direction="column" gap="xs" alignSelf="center" p="l">
+              <Flex gap="s">
+                <Text variant="heading" size="s">
+                  <DisplayAudio amount={newDelegationAmount} />
+                </Text>
+                <Text
+                  variant="heading"
+                  size="s"
+                  color="subdued"
+                  css={{ textDecoration: 'line-through' }}
+                >
+                  <DisplayAudio amount={delegates} />
+                </Text>
+              </Flex>
+              <Box>
+                <Text variant="body" size="m" strength="strong" color="subdued">
+                  {messages.newStakingAmount}
+                </Text>
+              </Box>
+            </Flex>
+            <Flex justifyContent="center">
+              <Button
+                isDisabled={
+                  !inputNumberValue ||
+                  inputNumberValue.isZero() ||
+                  status === Status.Loading
+                }
+                text={
+                  status === Status.Loading
+                    ? messages.loadingSubmit
+                    : messages.saveChanges
+                }
+                type={ButtonType.PRIMARY}
+                onClick={onConfirm}
               />
-            </div>
-          </div>
+            </Flex>
+          </Flex>
         </div>
-        <Button
-          text={isIncrease ? messages.increaseBtn : messages.decreaseBtn}
-          type={ButtonType.PRIMARY}
-          onClick={onClickConfirm}
-        />
-      </div>
-      <ConfirmTransactionModal
-        isOpen={isConfirmOpen}
-        onClose={onCloseConfirm}
-        onConfirm={onConfirm}
-        topBox={topBox}
-        bottomBox={bottomBox}
-        status={status}
-        error={error}
+      </Modal>
+      <ErrorModal
+        isOpen={isErrorModalOpen}
+        onClose={onCloseErrorModal}
+        message={!!error && error.includes('\n') ? error.split('\n')[0] : error}
+      />
+    </>
+  )
+}
+
+type ManageDelegationProps = {
+  delegates: BN
+  wallet: Address
+}
+export const ManageDelegation = ({
+  delegates,
+  wallet
+}: ManageDelegationProps) => {
+  const { isOpen, onClick, onClose } = useModalControls()
+  return (
+    <>
+      <Button
```

Comment:
Note - have TODO to disable this if there is a pending claim or pending decrease delegation tx